js/startup.js

```diff
@@ -10,14 +10,32 @@ var opts = stdio.getopt({
   'handler': {mandatory: true, args: 1, key: 'h', description: 'handler function to call'},
   'timeout': {mandatory: true, args: 1, key: 't', description: 'timeout for handler function'}
 });
-console.info(opts);
 
 var module = require(opts['module-path']);
 var server = http.createServer();
 
+server.on('listening', function () {
+  var local = server.address();
+
+  var host = (
+      local.family == "IPv6"
+      ? "[" + local.address + "]"
+      : local.address
+  );
+
+  server.url = 'http://' + host + ':' + local.port + '/';
+
+  console.info('Server running at', server.url, 'for ' + opts['module-path'] + ' / ' + opts.handler);
+});
+
```

Comment:
To make this more easily read by the ruby process, one could format the console.out into a more easily machine-readable format; something like:
```
server: http://localhost:6543/
module path: /var/path/somewhere/
handler: my_first_lambda
```